packages/vue/src/composables/useUtils.ts

```diff
@@ -1,13 +1,15 @@
-export const useAliases = (login_mechanisms: string[]): string[] => {
-  const defaultMechanisms = ['username', 'email', 'phone_number'];
+import { LoginMechanism, userNameAliasArray } from '@aws-amplify/ui';
 
+export const useAliases = (
+  login_mechanisms: LoginMechanism[]
+): LoginMechanism[] => {
   if (login_mechanisms) {
     if (login_mechanisms.length === 1 && login_mechanisms[0] === 'username') {
-      return defaultMechanisms;
+      return [...userNameAliasArray];
     } else {
```

Comment:
We don't need this block since the default return value is also `[...userNameAliasArray]`